.github/workflows/inspektor-gadget.yml

```diff
@@ -657,9 +660,15 @@ jobs:
       matrix:
         image:
           - name: "dnstester"
-            dockerfile-dir: "tools/dnstester"
+            context: "tools/dnstester"
+            dockerfile: "tools/dnstester/Dockerfile"
             platform: "linux/amd64,linux/arm64"
             filter-pattern: "tools/dnstester/*"
+          - name: "ebpf-builder"
+            context: "/home/runner/work/inspektor-gadget/inspektor-gadget"
+            dockerfile: "Dockerfiles/ebpf-builder.Dockerfile"
+            platform: "linux/amd64"
```

Comment:
Does it mean `ig image build` would not work on linux/arm64?

Reply:
I saw the [existing image](https://github.com/inspektor-gadget/inspektor-gadget/pkgs/container/ebpf-builder) was also only built for `linux/amd64` so I just used that but yes that would mean `ig image build` won't work on `linux/arm64` with builder image. Do you know if it fine to build it for `linux/arm64` ?

Reply:
I tried building it for `arm64` and running into following [error](https://github.com/mqasimsarfraz/inspektor-gadget/actions/runs/6493130779/job/17633458645#step:8:1181):
```
#9 33.43 E: Package 'gcc-multilib' has no installation candidate
#9 ERROR: process "/dev/.buildkit_qemu_emulator /bin/sh -c apt-get update \t&& apt-get install -y gcc-multilib lsb-release wget software-properties-common gnupg" did not complete successfully: exit code: 100
```

Reply:
I see, I opened a separate PR (it will also test if image is built fine for PR) for this, https://github.com/inspektor-gadget/inspektor-gadget/pull/2139 so we can track `arm64` alongside.

Reply:
It failed while building/pushing gadgets: https://github.com/inspektor-gadget/inspektor-gadget/actions/runs/6495111185/job/17642715987